pilot/docker/Dockerfile.proxyv2

```diff
@@ -14,7 +14,7 @@ COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
 
 RUN chown -R istio-proxy /var/lib/istio
 
-COPY istio-iptables.sh /usr/local/bin/istio-iptables
+COPY istio-iptables /usr/local/bin/istio-iptables
```

Comment:
@howardjohn I'd especially like feedback on this change. In 1.5+ it appears we switched to the go binary (which is what the validation code is in), however in 1.4 the proxyv2 image still runs the shell script. This is what was causing the test change I've been fighting.

Reply:
I get the issue, but changing this is far to invasive for a 1.4 patch release. We will need to come up with an alternative. Maybe just copy both in?

Reply:
That works for me, I thought it might be too big a change. I don't know how the behavior differs between the script and the binary. I'll add the script back in and add the binary in with a different name, then alter the injection template to call that for the validation container.